File: javascript/Day_2/Question_02.js
numbers = [10, -5, 3, -1, 0, 7, -8, 12, -3];

let count = 0;

const NumberChecker = () => {
  for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] % 2 === 0) {
      count += numbers[i];
    }
  }
  console.log(`total even number is ${count}`);
};

NumberChecker();

File: python/Day_2/Question_02.py
mixed_numbers = [10, -5, 3, -1, 0, 7, -8, 12, -3]

count = 0

for i in mixed_numbers:
    if(i % 2 == 0):
        print(i)
        count += i
print(f"total even number is: {count}")
    